zedd-app/package.json

```diff
@@ -89,85 +89,88 @@
     }
   },
   "dependencies": {
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
+    "@mui/icons-material": "^5.2.5",
+    "@mui/lab": "^5.0.0-alpha.64",
+    "@mui/material": "^5.2.8",
+    "@mui/styles": "^5.2.3",
     "@types/react-date-range": "^1.4.2",
     "react-date-range": "^1.4.0",
```

Comment:
Please move these dependencies to devDepenencies, as otherwise they will be copied into the built application unnecessarily